interfaces/builtin/repowerd.go

```diff
@@ -0,0 +1,307 @@
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
+)
+
+var repowerdPermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the repowerd service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+
+# HW alarms
+/dev/alarm rw,
+
+# Suspend
+/sys/power/** rw,
+
+# Backlight
+/sys/**/brightness rw,
+/sys/**/max_brightness r,
+/sys/**/type r,
+
+# DBus
+#include <abstractions/dbus-strict>
+
+dbus (send)
+  bus=system
+  path=/org/freedesktop/DBus
+  interface=org.freedesktop.DBus
+  member={GetId,RequestName,ReleaseName,GetConnectionUnixProcessID}
+  peer=(name=org.freedesktop.DBus),
+
+dbus (receive, send)
+  bus=system
+  interface=org.freedesktop.DBus.{Properties,Introspectable},
+
+dbus (bind)
+  bus=system
+  name=com.canonical.powerd,
+
+dbus (bind)
+  bus=system
+  name=com.canonical.Unity.Screen,
+
+dbus (receive, send)
+  bus=system
+  interface=com.canonical.powerd,
+
+dbus (receive, send)
+  bus=system
+  interface=com.canonical.Unity.Screen,
+
+# logind session
+dbus (send)
+  bus=system
+  path=/org/freedesktop/login1
+  interface=org.freedesktop.login1.Manager
+  member=GetSessionByPID
+  peer=(name=org.freedesktop.login1),
+dbus (send)
+  bus=system
+  path=/org/freedesktop/login1{,/seat/**,/session/**}
+  interface=org.freedesktop.DBus.Properties
+  member=Get{,All}
+  peer=(name=org.freedesktop.login1),
+dbus (receive)
+  bus=system
+  path=/org/freedesktop/login1
+  interface=org.freedesktop.login1.Manager
+  member={SessionAdded,SessionRemoved}
+  peer=(name=org.freedesktop.login1),
+dbus (receive)
+  bus=system
+  path=/org/freedesktop/login1{,/seat/**,/session/**}
+  interface=org.freedesktop.DBus.Properties
+  member=PropertiesChanged
+  peer=(name=org.freedesktop.login1),
+
+# logind power
+dbus (send)
+  bus=system
+  path=/org/freedesktop/login1
+  interface=org.freedesktop.login1.Manager
+  member={Hibernate,HybridSleep,Inhibit,PowerOff,Suspend}
+  peer=(name=org.freedesktop.login1),
+
+# Unity.Display
+dbus (send)
+  bus=system
+  interface=com.canonical.Unity.Display
+  path=/com/canonical/Unity/Display
+  member={TurnOn,TurnOff}
+  peer=(name=com.canonical.Unity.Display),
+
+# Unity.UserActivity
+dbus (receive)
+  bus=system
+  interface=com.canonical.Unity.UserActivity
+  path=/com/canonical/Unity/UserActivity
+  member=Activity,
+
+# Unity.PowerButton
+dbus (receive)
+  bus=system
+  interface=com.canonical.Unity.PowerButton
+  path=/com/canonical/Unity/PowerButton
+  member={Press,Release},
+dbus (send)
+  bus=system
+  interface=com.canonical.Unity.PowerButton
+  path=/com/canonical/Unity/PowerButton
+  member=LongPress,
+`)
+
+var repowerdConnectedPlugAppArmor = []byte(`
+# Description: Allow using repowerd service. Reserved because this gives
+#  privileged access to the repowerd service.
+# Usage: reserved
```

Comment:
Please remove 'Usage' and have only one space before 'privileged access to the repowerd service'